packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -91,8 +94,25 @@ const ProfilePage = () => {
       });
     },
   });
+  const { isLoading: isLoadingSSO, data: dataSSO } = useQuery('sso', async () => {
+    const { 
+      data: {
+        data
+      }
+    } = await get('/admin/providers/options');
+  
+    return data;
+  }, {
+    onSuccess() {},
+    onError() {
+      toggleNotification({
+        type: 'warning',
+        message: { id: 'Settings.permissions.users.sso.provider.error' },
+      });
+    },
+  });
 
-  const isLoading = status !== 'success';
+  const isLoading = isLoadingUser && isLoadingSSO;
```

Comment:
Shouldn't this be in a memo or something? What would happen if `isLoadingUser` or `isLoadingSSO` changes?

Reply:
No I think this is fine ... isLoading changes from false to true - memo would be overkill and probably slow things down even.